.gitignore

```diff
@@ -12,9 +12,13 @@ captures/
 .gradle/
 build/
 
+# CMake
+.cxx/
+
 # Local configuration file (sdk path, etc)
 local.properties
 repo/
+config/
```

Comment:
Why is this part of this PR ?

Reply:
Just to make sure I don't do the same mistake again...it will be anyway removed once I rebase with master